python/cudf_polars/cudf_polars/dsl/expressions/string.py

```diff
@@ -222,6 +223,23 @@ def do_evaluate(
                     plc.Scalar.from_py(None, plc.DataType(plc.TypeId.STRING)),
                 )
             )
+        elif self.name is StringFunction.Name.ZFill:
+            child, width = self.children
+            assert isinstance(width, Literal)
+            try:
+                pa.scalar(width.value, type=pa.uint64())
+            except OverflowError:
+                raise InvalidOperationError(
+                    f"conversion from `i32` to `u64` failed in column 'literal' for 1 out of 1 values: [{width.value}]"
+                ) from None
+            column = child.evaluate(df, context=context)
+            if width.value is None:
+                return Column(
+                    plc.interop.from_arrow(
+                        pa.array([None] * column.size, type=pa.string())
```

Comment:
```suggestion
                    plc.Column.from_scalar(plc.Scalar(None, plc.DataType(plc.TypeId.STRING), column.size)
```
(We're trying to move away from using pyarrow)